### ISSUE TITLE
csv dumps have “copy_of_” entries in the hasConnectionsWith column

### ISSUE THREAD

**Issue by paregorios** [isawnyu/pleiades-gazetteer#289]
From @rmhorne via email:

> It looks like our .csv dumps are still showing up with “copy_of_” entries in the hasConnectionsWith column:
> 
> - 383741  copy_of_106697791
> - 393421  copy_of_106697791
> - 109242  copy_of_109126
> - 109341  copy_of_109126
> - 109004  copy_of_114789
> - 981541  copy_of_128376
> - 246730  copy_of_246567
> - 383783  copy_of_383574
> - 502042016   copy_of_383609
> - 507729266   copy_of_383609
> - 502042016   copy_of_413107
> - 174900201   copy_of_413107
> - 413122  copy_of_413332
> - 422995  copy_of_423031
> - 423080  copy_of_423031
> - 356966898   copy_of_423038
> - 423025  copy_of_423038
> - 1356    copy_of_423080
> - 423031  copy_of_423080
> - 462492  copy_of_462068
> - 462492  copy_of_462131
> - 1043    copy_of_462168
> - 462492  copy_of_462204
> - 462503  copy_of_462204
> - 462492  copy_of_462412
> - 462492  copy_of_462487
> - 462193  copy_of_462487
> - 462492  copy_of_462489
> - 462492  copy_of_462503
> - 991353  copy_of_462503
> - 981549  copy_of_462503
> - 462492  copy_of_462506
> - 462492  copy_of_462513
> - 462492  copy_of_462527
> - 550406  copy_of_550404
> - 550650  copy_of_550651
> - 735295266   copy_of_550651
> - 837 copy_of_550812
> - 550759  copy_of_550812
> - 177302800   copy_of_550812
> - 570102  copy_of_570598
> - 570577  copy_of_570756
> - 442241741   copy_of_59969
> - 599668  copy_of_599799
> - 599777  copy_of_599799
> - 735295266   copy_of_599799
> - 837 copy_of_609292
> - 511363  copy_of_609292
> - 991381  copy_of_638753
> - 639116  copy_of_639139
> - 661388  copy_of_658381
> - 658560  copy_of_658381
> - 727172  copy_of_727097
> - 776172  copy_of_776235
> - 857352  copy_of_857350
> - 423025  copy_of_874299107
> - 628924  copy_of_874316
> - 912849  copy_of_893990
> - 20469   copy_of_98909
>
> This is not a problem with the  connectsWith column.

**Comment by davisagli** [comment]
This doesn't necessarily look like a bug to me; this lists the ids of the connected objects, and these are the ids that Plone uses if you copy something into a folder where that id already exists. It might be a feature request to customize how copied items get named.

**Comment by paregorios** [comment]
I think the implication is that we're getting information in the dumps about objects that are (presumably) in states other than "published" and the dumps are advertised as "published" data only. I'm going to spot check a few of these to see what states therein, then we'll undertake a discussion on whether this feature is desirable or not.

**Comment by paregorios** [comment]
Priority high because this looks like a permissions bleed and therefore constitutes data corruption. All download/exports should only make reference to PUBLISHED content.

**Comment by rmhorne** [comment]
I agree with @paregorios: I think there is an expectation that our .csv files reflect the current state of published / reviewed content, and that any working, copied, pending, or otherwise unfinished and unreviewed entries should be kept off of these.

**Comment by rmhorne** [comment]
This still appears to be an issue; the CSV dumps are still reading "copy_of" in the connection columns.

See https://pleiades.stoa.org/places/265984 in the CSV, which has "copy_of_270296"; There are many more examples.
